samples/ControlCatalog/App.xaml

```diff
@@ -60,7 +60,7 @@
     <Style Selector="Label.h3">
       <Setter Property="FontSize" Value="12" />
     </Style>
-  </Application.Styles>
+  </Application.Styles><!--
```

Comment:
Is it supposed to be commented?

Reply:
I think it should be removed completely since it breaks control catalog on linux framebuffer. Which would make people to think that framebuffer platform is broken